### ISSUE TITLE
Feature Request: Update existing entries on subsequent imports

### ISSUE THREAD

**Issue by brianyulke** [jampe/kp2bw#26]
It would be helpful to have an option to instruct kp2bw to update existing entries in BW rather than skip them if the entry in KeePass differs from that of BW.  This is helpful for a slow migration from KeePass to BW where the original KeePass database may still be in active use.

**Comment by jampe** [comment]
Good idea. I don't really have the time to work on this but if someone wants to implement this I'd welcome a PR ;-)

**Comment by jampe** [comment]
This feature is implemented in the maintained fork kjanat/kp2bw — it supports in-place updates (PUT instead of skip), UUID-based dedup, and content stamping. Check it out at github.com/kjanat/kp2bw
